CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 Changes
 =======
 
+0.2.0 (YYYY-MM-DD)
+------------------
+
+* New URL Processors:
+
+    * ``SubpathRemovalProcessor`` removes parts of a URL subpath based on the
+      given integer positions.
```

Comment:
Wondering if adding something like `given integer positions of the subpath` make this clearer.

Reply:
I think that makes it redundant.

I've updated it to `removes the subpaths of a URL based on its integer positions` instead.